templates/react/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: Build and Deploy
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+   
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    # https://github.com/marketplace/actions/setup-node-js-for-use-with-actions
+    - name: Setup Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    
+    - run: npm install
+    
+    - run: npm run build
+
+    - name: upload build artifacts
+      uses: actions/upload-artifact@v1
+      with:
+        name: build
+        path: build
+
+  publish:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      
+      - name: Download build artifacts
+        uses: actions/download-artifact@v1
+        with:
+          name: build
+
+      # https://github.com/opspresso/action-s3-sync
+      - name: Publish to AWS S3
+        uses: opspresso/action-s3-sync@master
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
where is the secrets struct being hydrated?

Reply:
Github Secrets. You just have to add them through their UI.

Reply:
Right that's why I couldn't find it anywhere  :+1: